master/docs/manual/cfg-reporters.rst

```diff
@@ -738,6 +738,7 @@ GerritStatusPush can send a separate review for each build that completes, or a
    :param builders: (optional) list of builders to send results for.
                     This method allows to filter results for a specific set of builder.
                     By default, or if builders is None, then no filtering is performed.
+   :param notify: (optional) control who gets notified by Gerrit once the status is posted.
```

Comment:
I think this would deserve to document which values are possible, probably point on the gerrit documentation, explaining that this correspond to the --notify values.